scripts/PKGBUILD

```diff
@@ -0,0 +1,23 @@
+# Maintainer: The-EDev <farook@the-e-dev.com>
+pkgname=crow
+pkgver=master
+pkgrel=1
+pkgdesc="A Fast and Easy to use C++ microframework for the web."
+arch=(any)
+url="https://crowcpp.org"
+license=('custom:BSD-3-Clause')
+depends=('boost>=1.64.0')
+optdepends=('openssl: HTTPS support' 'zlib: HTTP compression support' 'cmake: Choose this if you plan on using CMake for your Crow project')
+conflicts=("$pkgname-git")
+changelog='changelog.md'
+source=("https://github.com/CrowCpp/$pkgname/releases/download/v$pkgver/crow-v$pkgver.tar.gz")
+md5sums=('SKIP')
+sha256sums=('SKIP')
+
+package() {
+  echo "installing to \"$pkgdir/usr/local/\""
+  mkdir -p "$pkgdir/usr/local/include"
+  mkdir -p "$pkgdir/usr/local/lib"
+  cp -r "include" "$pkgdir/usr/local"
+  cp -r "lib" "$pkgdir/usr/local"
+}
```

Comment:
Does this really work? I can't make out where the include files are installed or where the cmake files are generated.

Reply:
Well the source package (the standalone package generated on release) has the cmake files, all this does is copy them to the same place cmake or deb would

Reply:
Ahh, I see. I didn't see that the source package was being used. But does the release archive contain the necessary CMake files?

Reply:
It contains everything the .deb contains, minus the metadata

Reply:
I think installing with CMake is mandatory. Depending on if ssl or compression is needed, the resulting CMake files are different. In the release archive, it seems like it is always both disabled.

Reply:
well this is another headache, because the deb package doesn't have these either, and IMO it doesn't make sense to restrict Crow to either using CMake exclusively or not at all..

Reply:
is there a way to have them be disabled by default and allow a project to enable them via CMake?

Reply:
Ok, I think I have a solution.
I thought of 2 ways:
```
find_package(Crow COMPONENTS ssl compression CONFIG REQUIRED)
target_link_libraries(app Crow::Crow)
```

```
set(CROW_ENABLE_SSL ON)
set(CROW_ENABLE_COMPRESSION ON)
find_package(Crow CONFIG REQUIRED)
target_link_libraries(app Crow::Crow)
```

I could also add both ways to enable a feature. What do you think?

Reply:
I was thinking of something like the second solution

Reply:
Ok, I'll spin up a PR to get this resolved. 👍